__tests__/all_spec.js

```diff
@@ -144,12 +144,17 @@ names.forEach((name) => {
           text: 'Pending...',
           timeout: 2 * 1000,
         });
+        // Make sure that while isPending true, previous state displayed
+        await expect(page.evaluate(() => document.querySelector('#maincount').innerHTML)).resolves.toBe('0');
```

Comment:
Isn't it `#mainCount` or it's case-insensitive?

Reply:
Sorry about that.   Yes it is case-insensitive because there isn't a <!doctype html> in index.html, causing quirks mode to be in effect.  That is why it worked when it should have failed all test 1s when I first added it.